plugin-server/tests/main/ingestion-queues/each-batch.test.ts

```diff
@@ -162,46 +161,6 @@ describe('eachBatchX', () => {
             await eachBatchAppsOnEventHandlers(createKafkaJSBatch(clickhouseEvent), queue)
             expect(runOnEvent).not.toHaveBeenCalled()
         })
-        it('parses elements when useful', async () => {
```

Comment:
we should keep this test

Reply:
We do :D but we moved the code so I had to move the test